src/test/crimson/test_errorator.cc

```diff
@@ -6,9 +6,9 @@
 #include "crimson/common/errorator.h"
 #include "crimson/common/log.h"
 
-struct do_until_test_t : public seastar_test_suite_t {
```

Comment:
nit: typo in the commit message:

`so we can reuse this test suite for more erroratoer related tests.` -> `so we can reuse this test suite for more errorator related tests.`

Reply:
thanks. fixed and repushed.